src/geom/rectbin.jl

```diff
@@ -1,13 +1,7 @@
-### should refactor to RectangleGeometry with Identity as the default Stat.
-struct RectangularBinGeometry <: Gadfly.GeometryElement
-    default_statistic::Gadfly.StatisticElement
-    tag::Symbol
-end
-
 function RectangularBinGeometry(
         default_statistic::Gadfly.StatisticElement=Gadfly.Stat.rectbin();
         tag=empty_tag)
-    RectangularBinGeometry(default_statistic, tag)
+    RectangularGeometry(default_statistic, tag)
 end
```

Comment:
is there a reason to not delete `RectangularBinGeometry` too?  `rectbin` could then simply be defined as `RectagularyGeometry(Stat.rectbin())`.

Reply:
For the moment, just the `theme_bar_spacing`. I recognised the same possibility and tried to handle the `theme_bar_spacing` in `Stat.rectbin`, but got knocked back with an incompatible type issue.

Reply:
`RectangularBinGeometry` deleted in https://github.com/GiovineItalia/Gadfly.jl/pull/1263/commits/2c5823054be5581fa74a349828b2973ce1d06a88. This will likely get squashed into another commit.

Reply:
Refactoring was reverted to allow `RectangularBinGeometry` and `RectangularGeometry` to evolve separately.